common/source/docs/common-ac2_simple_geofence.rst

```diff
@@ -59,29 +59,68 @@ switches.  Like with the earlier fences, another fence is erected 20m
+
++------------+--------------------+
++FENCE_ACTION|  Plane             +
++============+====================+
++     0      |   Report Only      +
++------------+--------------------+
++     1      | RTL                +
++------------+--------------------+
++     2      | Guided to return   +
++            | point              +
++------------+--------------------+
++     3      |  Guided to return  +
++            |  point with pilot  +
++            |  throttle control  +
++------------+--------------------+
+
+The guided return altitude during the return to the return point after breach, if this action is selected, is determined by :ref:`FENCE_RET_ALT<FENCE_RET_ALT>` . If either :ref:`FENCE_ACTION<FENCE_ACTION>` =2 or 3 is used. A fence return point **MUST** be setup or unpredictable action will occur (see video below). And the return point can be replaced by the nearest rally point as a destination, if :ref:`FENCE_RET_RALLY<FENCE_RET_RALLY>` is enabled.
+
+
 Enabling the Fence in Mission Planner
 =====================================
 
-.. image:: ../../../images/Fence_MPSetup.png
-    :target: ../_images/Fence_MPSetup.png
+To enable Fences, go to the Mission Planner full parameter list (CONFIG->Full Parameter Tree), search for items with ``FENCE_``:
+
+.. image:: ../../../images/fence_enable.png
+    :target: ../_images/fence_enable.png
 
-The Fence can be set-up by doing the following:
+Plane Parameter List (Copter/Rover have fewer)
 
--  Connect your autopilot to the Mission Planner
--  Go to the **CONFIG/GeoFence** screen
--  Enable the FENCE: :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1.
+-  Set :ref:`FENCE_ACTION<FENCE_ACTION>` = to whatever you wish for a breach action. These will vary depending on vehicle type.
+-  Set :ref:`FENCE_ALT_MAX<FENCE_ALT_MAX>` = to the altitude limit you want (in meters). This is unavailable in Rover. A value of 0 disables this limit.
+-  Set :ref:`FENCE_ENABLE<FENCE_ENABLE>` =1 to enable FENCEs.
+-  Set :ref:`FENCE_MARGIN<FENCE_MARGIN>` = to the distance from the fence horizontal boundary the vehicle must maintain in order to prevent a breach.
+-  Set :ref:`FENCE_RADIUS<FENCE_RADIUS>` to the maximum distance from HOME you want (in meters). This should normally be at least 50m. This value must be larger than :ref:`FENCE_MARGIN<FENCE_MARGIN>` and greater than 30m.
+-  :ref:`FENCE_TOTAL<FENCE_TOTAL>` should not be changed manually and indicates the number of inclusion/exclusion fence points stored. (See :ref:`common-polygon_fence`)
+-  Set :ref:`FENCE_TYPE<FENCE_TYPE>` = default of "ALL" is fine and will allow you to set up an additional :ref:`common-polygon_fence` , unless you want only a cylindrical fence to be active if a polygon fence has also been loaded. Then you could select "Altitude" or "Circle" or both.
```

Comment:
Default is not set to `ALL` - ever. 
Rover defaults to Circle + Polygon.
Plane defaults to Polygon.
All others default to Alt Max + Tin Can + Polygon.

We still have ALT Min available, that's not used by default

Reply:
that was historical in the previous document....changed